src/Stats.chpl

```diff
@@ -0,0 +1,23 @@
+module Stats {
+    use AryUtil;
+
+    proc mean(ar1: [?aD1] ?t): real throws {
+        return (+ reduce ar1:real) / aD1.size:real;
+    }
+
+    proc variance(ar1: [?aD1] ?t, ddof: int): real throws {
+        return (+ reduce ((ar1:real - mean(ar1)) ** 2)) / (aD1.size - ddof):real;
+    }
+
+    proc std(ar1: [?aD1] ?t, ddof: int): real throws {
+        return sqrt(variance(ar1, ddof));
+    }
```

Comment:
This is not an issue just me being picky. In both these functions, my brain wanted to look for `ar2` and `aD2` because of the naming. Since there are only 1 array being passed to these functions, could we change `ar1` and `aD1` to `ar` and `aD`.  I think this will read a little easier, but is not a deal breaker.

Reply:
yeah that's a good idea!

Reply:
Just being picky again, you updated 
`proc std(ar1: [?aD1] ?t, ddof: int): real throws {`
to 
`proc std(ar: [?aD] ?t1, ddof: int): real throws {`
Can we remove the `1` from `?t1` so it matches the definitions of `mean` and `variance`?